src/Admin/AbstractAdmin.php

```diff
@@ -3029,7 +3029,7 @@ protected function buildForm()
 
             $value = $propertyAccessor->getValue($object, $propertyPath);
 
-            if (\is_array($value) || ($value instanceof \Traversable && $value instanceof \ArrayAccess)) {
+            if (\is_array($value) || $value instanceof \ArrayAccess) {
```

Comment:
`\ArrayAccess` does not imply `\Traversable` does it? Why remove that part of the check?

Reply:
Because the fact an object can be traversed doesn't imply that it can be handled as an array, using accessors (`[]`) by instance.

Reply:
Likewise, the fact that an object can be handled as an array, using accessors does not imply that it can be traversed. If you remove that check, `$value` can no longer be traversed, but maybe that's something that does not happen? Given the name of `setValue`, I guess it doesn't.

Reply:
I guess you're right. But reading your comment, I figured out that I was interpreting this change as a bugfix for a syntax error by mistake (I was reading the `&&` as `||`).
That said, I don't know what this change can imply from a behavioral point of view.
I'd prefer to revert this line and just keep the pedantic change.
I'll be waiting for your concerns.